src/components/PopupTrigger.js

```diff
@@ -41,27 +41,34 @@ export default class PopupTrigger extends Component {
     this.setState({ open: false })
   }
 
+  setOpen(v) {
+    const { onToggle } = this.props
+    this.setState({ open: v })
+    onToggle && onToggle(v)
+  }
+
   render() {
-    const { children, content, position, align, closeOnClick } = this.props
+    const { children, content, position, align, closeOnClick, open: forceOpen } = this.props
     const { open } = this.state
     const child = Children.only(children)
+    const shouldShow = forceOpen === undefined ? open : forceOpen
     return h(Fragment, [
       cloneElement(child, {
         id: this.id,
         className: `${child.props.className || ''} ${this.id}`,
         onClick: (...args) => {
           child.props.onClick && child.props.onClick(...args)
-          this.setState({ open: !open })
+          this.setOpen(!shouldShow)
```

Comment:
since setOpen is only used by the in this method, consider defining it locally to the render function rather than on the component itself.